### ISSUE TITLE
Add `inplace_view` tag for `resize_`

### ISSUE THREAD

**Issue by anjali411** [pytorch/pytorch#78759]
### 🐛 Describe the bug

as per the title 

### Versions

master

**Comment by eellison** [comment]
Can we just special case `resize_` in the codegen logic and add the tag ? it kind of limits using tags if they're not fully general

**Comment by anjali411** [comment]
cc. @bdhirsh

**Comment by bdhirsh** [comment]
Yeah, special casing the codegen so we can add the tag to resize_ sounds good to me. Functionalization already has to special case resize_ anyway.

**Comment by eellison** [comment]
Bump on this - `inplace_view` is starting in a bunch of places and it isn't catching `resize_` bc of one error in codegen
